package.json

```diff
@@ -11,16 +11,26 @@
   "bugs": {
     "url": "https://github.com/ng-bootstrap/schematics/issues"
   },
+  "keywords": [
+    "angular",
+    "ng-bootstrap"
+  ],
   "homepage": "https://github.com/ng-bootstrap/schematics#readme",
   "scripts": {
-    "clean": "rm -rf dist",
-    "build": "rm -rf dist && tsc && node ./create_package.js && cp src/collection.json dist"
+    "test": "jest",
+    "clean": "rm -Rf dist",
+    "build": "yarn clean && mkdir -p dist && node scripts/create_package.js && tsc -p tsconfig.json"
+  },
+  "dependencies": {
+    "@angular-devkit/core": "^7.0.6",
+    "@angular-devkit/schematics": "^7.0.6",
+    "@schematics/angular": "^7.0.6",
```

Comment:
I wonder if there any relation between Angular version, CLI version and our dependencies in `dist/package.json`:

```json
"dependencies": {
    "@angular-devkit/core": "^7.0.6",
    "@angular-devkit/schematics": "^7.0.6",
    "@schematics/angular": "^7.0.6",
    "typescript": "~3.1.4"
  }
```